### PR TITLE
SwapShareBackup : Fix' backup delete from user' issue in cleanup

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
Deleting backups from user before deleting all backups from admin
**Which issue(s) this PR fixes** (optional)
Closes #
https://portworx.atlassian.net/browse/PA-1685
**Special notes for your reviewer**:
log : https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/2949/consoleFull

### CODE REVIEW

**Review by ak-px — APPROVED** [review]
LGTM

File: tests/backup/backup_share_test.go [hunk 1]
```diff
@@ -4134,6 +4129,14 @@ var _ = Describe("{SwapShareBackup}", func() {
 			log.FailOnError(err, fmt.Sprintf("Error deleting backup - %s for user - %s", backupName, users[i]))
 		}
 
+		Step("Delete all synced backups from the admin", func() {
```
Comment (kshithijiyer-px) [inline tests/backup/backup_share_test.go:4132]: Can we remove this?

Comment (snigdha-px) [reply tests/backup/backup_share_test.go]: done

File: tests/backup/backup_share_test.go [hunk 2]
```diff
@@ -4025,8 +4026,9 @@ var _ = Describe("{SwapShareBackup}", func() {
 
 				backupDriver := Inst().Backup
 				backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+				log.FailOnError(err, "Failed to get backup UID for backup %s for user %s", backupName, user)
 				backupUIDList = append(backupUIDList, backupUID)
-				log.FailOnError(err, "Failed getting backup uid for backup name %s", backupName)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Getting backup uid %s of backup: %s for user %s", backupUID, backupName, user))
```
Comment (mkoppal-px) [inline tests/backup/backup_share_test.go:4031]: Please remove this

Comment (snigdha-px) [reply tests/backup/backup_share_test.go]: done

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**